### DISCUSSION TITLE
What are the differences between Lora file from Kohya and Autotrain?

### DISCUSSION THREAD

**Discussion by dvelm** [huggingface/autotrain-advanced#539] [Q&A]
The lora from Kohya are visible in Stable Diffusion Forge , while lora from Autotrain are not usable in forge. The autotrain lora have small dimensions than Kohya lora. Does autotrain compress them? How to avoid the compression? In fooocus both Kohya and autotrain lora are visible, in forge kohya. How to have the same output?

**Comment by urostrickovic** [comment]
The problem is that Kohya uses webui to train loras, but Autotrain uses dreambooth, so loras are not compatible in Stable diffusion. Im trying to find answer to is also. Check convert dreambooth lora to webui

**Reply by dvelm** [reply]
With webui can be done dreambooth too. It's the way it saves the safetensor that does conversions to make the file smaller and apply some optimization I think. On fooocus it works good, in others it's not seen
